examples/cpp/put-get-remove/startserver.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -f
```

Comment:
You didn't take any input on the start script. Please see comments.

Reply:
I have now, I didn't see the notes before for some reason.